### DISCUSSION TITLE
Not able to see netbox home page on http

### DISCUSSION THREAD

**Discussion by alejandro-logi** [netbox-community/netbox#10068] [Q&A]
Hello,

I'm installing netbox from zero following below guide but I'm getting an error.
https://github.com/netbox-community/netbox/blob/v2.10.2/docs/installation/3-netbox.md

When I get to the last step where you start the local server and try to reach it via http, I get the attached error.
![error_netbox](https://user-images.githubusercontent.com/111561503/185564395-69301768-f064-414a-90e2-eecafa09986c.png)

I've reviewed the whole guide and I'm not able to find the error.

Thanks for any help.
Best regards.

**Comment by richbibby** [comment]
Hi. Looks like you are following the install guide for an older version (v2.10.2). The current version is v3.3.0. 

I would suggest you try the official docs for the latest install guide: https://docs.netbox.dev/en/stable/installation/

Rich

**Comment by abhi1693** [comment]
> Hello,
> 
> I'm installing netbox from zero following below guide but I'm getting an error.
> https://github.com/netbox-community/netbox/blob/v2.10.2/docs/installation/3-netbox.md
> 
> When I get to the last step where you start the local server and try to reach it via http, I get the attached error.
> ![error_netbox](https://user-images.githubusercontent.com/111561503/185564395-69301768-f064-414a-90e2-eecafa09986c.png)
> 
> I've reviewed the whole guide and I'm not able to find the error.
> 
> Thanks for any help.
> Best regards.

Did you try the suggestion in the screenshot as well? Because that is the solution for the issue I see.
